### ISSUE TITLE
Xgboost prediction intervals 

### ISSUE THREAD

**Issue by Shafi2016** [benoitdescamps/benoit-descamps-blogs#2]
Hello Benoit, Thanks for providing this nice codes/examples for getting the XGBoost Prediction interval. 

I have 27 features with one target variable. But to make it reproducible I have posting codes for Boston housing data. The picture is given below for the issue. As I add more variables, the range for X-axis also goes up. I do not know how to solve this problem. 

from sklearn.datasets import load_boston
boston_dataset = load_boston()
df = pd.DataFrame(boston_dataset.data, columns=boston_dataset.feature_names)
df['MEDV'] = boston_dataset.target
df.tail()
X_train = df.iloc[1:300, 0:12]
X_test = df.iloc[301:505 , 0:12]
y_train = df.iloc[1:300 , 13]
y_test = df.iloc[301:505 , 13]
y_test.head(2)

![image](https://user-images.githubusercontent.com/56795978/77199688-a0f61100-6abf-11ea-9027-e38a36419d8d.png)
